web/client/actions/mapimport.js

```diff
@@ -0,0 +1,75 @@
+/**
+ * Copyright 2016, GeoSolutions Sas.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+const SET_LAYERS = 'IMPORT::SET_LAYERS';
```

Comment:
Why double colon?

Reply:
Good catch, I will fix it together with shapefile stuff